src/test.js

```diff
@@ -571,3 +571,27 @@ function skip( testName ) {
 
 	test.queue();
 }
+
+// Will be exposed as QUnit.only
+function only( testName, expected, callback, async ) {
+	var newTest;
+
+	if ( arguments.length === 2 ) {
+		callback = expected;
+		expected = null;
+	}
+
+	newTest = new Test({
+		testName: testName,
+		expected: expected,
+		async: async,
+		callback: callback
+	});
+
+	QUnit.config.queue.length = 0;
+	QUnit.config.filter = newTest.module.name + ": " + newTest.testName;
+
+	newTest.queue();
+
+	QUnit.only = QUnit.test; // First only wins, ignore subsequent onlys
```

Comment:
[Code style](http://contribute.jquery.org/style-guide/js/#comments) issue:

- Comments are always preceded by a blank line
- Single line comments go over the line they refer to